### ISSUE TITLE
pogcal import broken because of layout change

### ISSUE THREAD

**Issue by lepinkainen** [Flexget/Flexget#1198]
### Expected behaviour:
Shows are imported from http://www.pogdesign.co.uk/cat/
### Actual behaviour:
Not imported :)

```
Error during input plugin pogcal: Error retrieving source: 500 Server Error: Internal Server Error for url: http://www.pogdesign.co.uk/cat/show-select
Input pogcal did not return anything
```

**Comment by lepinkainen** [comment]
It seems that this is not a straight-forward fix. 

If I'm interpreting things correctly the damn idiots at pogcal are setting the show-select list via javascript after loading OR the login system has changed.

Will look into this later tonight.

**Comment by gazpachoking** [comment]
Looks like @pilluli is on it. #1199

**Comment by lepinkainen** [comment]
Possibly fixed with PR #1199, I merged it and will close this if it is so.

**Comment by lepinkainen** [comment]
#1199 fixed this issue with minor tweaks seems to work well enough now.
